config/config.exs

```diff
@@ -0,0 +1,5 @@
+use Mix.Config
+
+config :discourse,
```

Comment:
is this config here correct/required?

Reply:
Is indeed correct and serves two purposes:
1. Example
2. Values for tests.

Reply:
I see, i'm more in favour of moving it in `config/test.exs` since they only have a meaning there but not a big issue 
(also it would be nice to have the example configuration in the README as well)